INFO 2025-01-26 16:07:10 _internal.py: 224]  * Restarting with watchdog (windowsapi)
INFO 2025-01-26 16:07:17 utils.py: 160] NumExpr defaulting to 8 threads.
WARNING 2025-01-26 16:07:18 _internal.py: 224]  * Debugger is active!
INFO 2025-01-26 16:07:18 _internal.py: 224]  * Debugger PIN: 162-349-206
INFO 2025-01-26 16:50:10 utils.py: 160] NumExpr defaulting to 8 threads.
INFO 2025-01-26 16:50:10 _internal.py: 224] [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
INFO 2025-01-26 16:50:10 _internal.py: 224] [33mPress CTRL+C to quit[0m
INFO 2025-01-26 16:50:10 _internal.py: 224]  * Restarting with watchdog (windowsapi)
INFO 2025-01-26 16:50:16 utils.py: 160] NumExpr defaulting to 8 threads.
WARNING 2025-01-26 16:50:16 _internal.py: 224]  * Debugger is active!
INFO 2025-01-26 16:50:16 _internal.py: 224]  * Debugger PIN: 162-349-206
INFO 2025-01-26 16:50:28 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\site-packages\\flask\\globals.py', reloading
INFO 2025-01-26 16:50:28 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\site-packages\\flask\\json\\__init__.py', reloading
INFO 2025-01-26 16:50:28 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\site-packages\\flask\\wrappers.py', reloading
INFO 2025-01-26 16:50:28 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\site-packages\\werkzeug\\wrappers\\__init__.py', reloading
INFO 2025-01-26 16:50:28 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\site-packages\\werkzeug\\wrappers\\response.py', reloading
INFO 2025-01-26 16:50:28 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\site-packages\\werkzeug\\sansio\\response.py', reloading
INFO 2025-01-26 16:50:28 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\site-packages\\flask\\typing.py', reloading
INFO 2025-01-26 16:50:28 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\site-packages\\werkzeug\\wrappers\\request.py', reloading
INFO 2025-01-26 16:50:28 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\site-packages\\werkzeug\\sansio\\request.py', reloading
INFO 2025-01-26 16:50:28 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\site-packages\\werkzeug\\utils.py', reloading
INFO 2025-01-26 16:50:30 _internal.py: 224]  * Restarting with watchdog (windowsapi)
INFO 2025-01-26 16:50:36 utils.py: 160] NumExpr defaulting to 8 threads.
WARNING 2025-01-26 16:50:37 _internal.py: 224]  * Debugger is active!
INFO 2025-01-26 16:50:37 _internal.py: 224]  * Debugger PIN: 162-349-206
INFO 2025-01-26 16:51:41 utils.py: 160] NumExpr defaulting to 8 threads.
INFO 2025-01-26 16:51:52 utils.py: 160] NumExpr defaulting to 8 threads.
INFO 2025-01-26 16:51:52 utils.py: 160] NumExpr defaulting to 8 threads.
INFO 2025-01-26 16:51:52 utils.py: 160] NumExpr defaulting to 8 threads.
INFO 2025-01-26 16:52:02 utils.py: 160] NumExpr defaulting to 8 threads.
INFO 2025-01-26 16:52:02 utils.py: 160] NumExpr defaulting to 8 threads.
WARNING 2025-01-26 16:53:49 easyocr.py: 80] Neither CUDA nor MPS are available - defaulting to CPU. Note: This module is much faster with a GPU.
INFO 2025-01-26 16:53:52 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\multiprocessing\\connection.py', reloading
INFO 2025-01-26 16:53:52 _internal.py: 224] 127.0.0.1 - - [26/Jan/2025 16:53:52] "POST /detect HTTP/1.1" 200 -
INFO 2025-01-26 16:53:53 _internal.py: 224]  * Restarting with watchdog (windowsapi)
INFO 2025-01-26 16:54:00 utils.py: 160] NumExpr defaulting to 8 threads.
WARNING 2025-01-26 16:54:00 _internal.py: 224]  * Debugger is active!
INFO 2025-01-26 16:54:00 _internal.py: 224]  * Debugger PIN: 162-349-206
INFO 2025-01-26 17:30:09 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Scripts\\conda-script.py', reloading
INFO 2025-01-26 17:30:11 _internal.py: 224]  * Restarting with watchdog (windowsapi)
INFO 2025-01-26 17:30:16 utils.py: 160] NumExpr defaulting to 8 threads.
WARNING 2025-01-26 17:30:17 _internal.py: 224]  * Debugger is active!
INFO 2025-01-26 17:30:17 _internal.py: 224]  * Debugger PIN: 162-349-206
INFO 2025-01-26 17:32:38 _internal.py: 224]  * Detected change in 'D:\\workspace\\hb_projects\\danger_detection\\face_sdk\\core\\model_loader\\face_alignment\\FaceAlignModelLoader.py', reloading
INFO 2025-01-26 17:32:38 _internal.py: 224]  * Detected change in 'd:\\workspace\\hb_projects\\danger_detection\\face_sdk\\core\\model_loader\\face_alignment\\FaceAlignModelLoader.py', reloading
INFO 2025-01-26 17:32:38 _internal.py: 224]  * Detected change in 'D:\\workspace\\hb_projects\\danger_detection\\face_sdk\\core\\model_handler\\face_alignment\\FaceAlignModelHandler.py', reloading
INFO 2025-01-26 17:32:38 _internal.py: 224]  * Detected change in 'd:\\workspace\\hb_projects\\danger_detection\\face_sdk\\core\\model_handler\\face_alignment\\FaceAlignModelHandler.py', reloading
INFO 2025-01-26 17:32:38 _internal.py: 224]  * Detected change in 'D:\\workspace\\hb_projects\\danger_detection\\face_sdk\\core\\model_loader\\face_recognition\\FaceRecModelLoader.py', reloading
INFO 2025-01-26 17:32:38 _internal.py: 224]  * Detected change in 'd:\\workspace\\hb_projects\\danger_detection\\face_sdk\\core\\model_loader\\face_recognition\\FaceRecModelLoader.py', reloading
INFO 2025-01-26 17:32:38 _internal.py: 224]  * Detected change in 'D:\\workspace\\hb_projects\\danger_detection\\face_sdk\\core\\model_handler\\face_recognition\\FaceRecModelHandler.py', reloading
INFO 2025-01-26 17:32:38 _internal.py: 224]  * Detected change in 'd:\\workspace\\hb_projects\\danger_detection\\face_sdk\\core\\model_handler\\face_recognition\\FaceRecModelHandler.py', reloading
INFO 2025-01-26 17:32:38 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\site-packages\\torchvision\\__init__.py', reloading
INFO 2025-01-26 17:32:38 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\site-packages\\torch\\jit\\__init__.py', reloading
INFO 2025-01-26 17:32:38 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\site-packages\\torch\\jit\\_trace.py', reloading
INFO 2025-01-26 17:32:38 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\site-packages\\torch\\_jit_internal.py', reloading
INFO 2025-01-26 17:32:39 _internal.py: 224]  * Detected change in 'D:\\workspace\\hb_projects\\danger_detection\\face_sdk\\utils\\lms_trans.py', reloading
INFO 2025-01-26 17:32:39 _internal.py: 224]  * Detected change in 'd:\\workspace\\hb_projects\\danger_detection\\face_sdk\\utils\\lms_trans.py', reloading
INFO 2025-01-26 17:32:39 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\logging\\config.py', reloading
INFO 2025-01-26 17:32:40 _internal.py: 224]  * Restarting with watchdog (windowsapi)
INFO 2025-01-26 17:32:46 utils.py: 160] NumExpr defaulting to 8 threads.
WARNING 2025-01-26 17:32:46 _internal.py: 224]  * Debugger is active!
INFO 2025-01-26 17:32:46 _internal.py: 224]  * Debugger PIN: 162-349-206
INFO 2025-01-26 17:33:31 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\logging\\__init__.py', reloading
INFO 2025-01-26 17:33:31 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\site-packages\\_pytest\\cacheprovider.py', reloading
INFO 2025-01-26 17:33:31 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\site-packages\\_pytest\\fixtures.py', reloading
INFO 2025-01-26 17:33:31 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\site-packages\\_pytest\\config\\__init__.py', reloading
INFO 2025-01-26 17:33:31 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\site-packages\\_pytest\\capture.py', reloading
INFO 2025-01-26 17:33:31 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\site-packages\\_pytest\\doctest.py', reloading
INFO 2025-01-26 17:33:31 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\site-packages\\_pytest\\compat.py', reloading
INFO 2025-01-26 17:33:31 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\site-packages\\_pytest\\junitxml.py', reloading
INFO 2025-01-26 17:33:31 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\site-packages\\_pytest\\legacypath.py', reloading
INFO 2025-01-26 17:33:31 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\site-packages\\_pytest\\logging.py', reloading
INFO 2025-01-26 17:33:31 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\site-packages\\_pytest\\monkeypatch.py', reloading
INFO 2025-01-26 17:33:31 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\site-packages\\_pytest\\recwarn.py', reloading
INFO 2025-01-26 17:33:31 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\site-packages\\_pytest\\tmpdir.py', reloading
INFO 2025-01-26 17:33:33 _internal.py: 224]  * Restarting with watchdog (windowsapi)
INFO 2025-01-26 17:33:38 utils.py: 160] NumExpr defaulting to 8 threads.
WARNING 2025-01-26 17:33:39 _internal.py: 224]  * Debugger is active!
INFO 2025-01-26 17:33:39 _internal.py: 224]  * Debugger PIN: 162-349-206
INFO 2025-01-26 17:33:48 _internal.py: 224]  * Detected change in 'd:\\workspace\\hb_projects\\danger_detection\\face_sdk\\api_usage\\face_pipline.py', reloading
INFO 2025-01-26 17:33:48 _internal.py: 224]  * Detected change in 'D:\\workspace\\hb_projects\\danger_detection\\face_sdk\\api_usage\\face_pipline.py', reloading
INFO 2025-01-26 17:33:51 _internal.py: 224]  * Restarting with watchdog (windowsapi)
INFO 2025-01-26 17:33:57 utils.py: 160] NumExpr defaulting to 8 threads.
WARNING 2025-01-26 17:33:57 _internal.py: 224]  * Debugger is active!
INFO 2025-01-26 17:33:57 _internal.py: 224]  * Debugger PIN: 162-349-206
INFO 2025-01-26 17:34:13 _internal.py: 224]  * Detected change in 'D:\\anaconda3\\Lib\\abc.py', reloading
